src/main/resources/templates/index.html

```diff
@@ -1,39 +1,41 @@
 <!DOCTYPE html>
 
 <html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
       ng-app="healthBam">
 
     <head>
 
-        <meta charset="UTF-8">
+        <meta charset="UTF-8" />
```

Comment:
Thymeleaf is very complainful :)

It wants xml (or maybe xhtml), so I had to make sure all tags were closed and all attributes had a =.